src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php

```diff
@@ -397,7 +397,15 @@ public function exportAction(OrderFilters $filters)
 
             $data[] = $item;
         }
-
+        $this->dispatchHook(
+            'actionExportOrder',
+            [
+                'header' => &$headers,
+                'data' => &$data,
+                'orderGrid' => $orderGrid,
+                'isB2bEnabled' => $isB2bEnabled,
+            ]
+        );
```

Comment:
Please don't forget the extra line before the return statement. This is a code style rule.

You can test the code style yourself before you push with this command line:
`php vendor/bin/php-cs-fixer fix --dry-run --diff`